Checklists/README.md

```diff
@@ -14,3 +14,60 @@ The MSTG version element (Dashboard - row 13) in the excel file represent the ve
 Note that due to the continuous updating and restructuring of the MSTG, the links of different versions are not all identical. And changing the mstg version (Dashboard - row 13) may break the links in the excel file.
 
 Newer and older versions of the Excel are released on a regular basis and can be found at [the release page](https://github.com/OWASP/owasp-mstg/releases "Releases").
+
+## Files Hash
+
```

Comment:
Can you please explain what the hash means so people understand why it is important? (E.g. that we vet the security of the excel, but people should only trust the excel when the hashes match).